pkg/errors/errorMessage.go

```diff
@@ -206,6 +206,7 @@ var (
 	GALASA_ERROR_DELETE_PROPERTY_RESPONSE_PARSING        = NewMessageType("GAL1117E: The delete operation failed. Unable to process the error information returned from the server.", 1117, STACK_TRACE_WANTED)
 	GALASA_ERROR_RESOURCE_RESPONSE_PARSING               = NewMessageType("GAL1118E: The resource operation failed. Unable to process the error information returned from the server. Reason: '%s'", 1118, STACK_TRACE_WANTED)
 	GALASA_ERROR_RESOURCE_RESP_UNAUTHORIZED_OPERATION    = NewMessageType("GAL1119E: The server thinks you are unauthorized to perform this operation.", 1119, STACK_TRACE_NOT_WANTED)
+	GALASA_ERROR_COMMAND_NOT_FOUND                       = NewMessageType("GAL1120E: '%s' not found in command collection", 1120, STACK_TRACE_NOT_WANTED)
```

Comment:
Better to relate this to something the user understands. By all means log a message in the message log saying what's gone wrong,  but the error text should be something simple like "GAL1120E: Program logic error. Collect a log using the '--log' option and send to the Galasa development team."